docs/azure-openshift.md

```diff
@@ -24,78 +23,43 @@ $ SUBSCRIPTION_ID=$(oc get secrets azure-credentials  -n kube-system  --template
 $ TENANT_ID=$(oc get secrets azure-credentials  -n kube-system  --template={{.data.azure_tenant_id}} | base64 -d)
 ```
 
-4. Login to azure with base64 decoded values you get from above
-
+4. Login to Azure with base64 decoded values you get from above:
 ```bash
 $ az login --service-principal -u "${CLIENT_ID}" -p "${CLIENT_SECRET}" --tenant "${TENANT_ID}"
 ```
 
-5. Get the routes to check the domain
+5. Get the routes to check your cluster's domain (everything after `apps`):
 ```bash
-oc get routes --all-namespaces | grep console
+$ oc get routes --all-namespaces | grep console
 openshift-console          console             console-openshift-console.apps.test-azure.qe.azure.devcluster.openshift.com                       console             https   reencrypt/Redirect     None
 openshift-console          downloads           downloads-openshift-console.apps.test-azure.qe.azure.devcluster.openshift.com                     downloads           http    edge/Redirect          None
 ```
 
-6. Get the dns zone list w.r.t your resource group.
+6. Get the list of dns zones to find the one which corresponds to the previously found route’s domain:
 ```bash
 $ az network dns zone list --resource-group "${RESOURCE_GROUP}"
-[]
 ```
-Initially you will see nothing. So you will have to create a zone for your resource group.
 
-7. Create a dns zone for you resourcegroup.
+7. Create [ExternalDNS CR](https://github.com/openshift/external-dns-operator/blob/main/config/samples/azure/operator_v1alpha1_externaldns_openshift.yaml) as follows:
```

Comment:
Have you removed step 7 as we have introduced azure-private-dns ?

Reply:
I removed it because we use the cluster's zone in this doc, that's why we listed routes before, to get the cluster's domain.
This doc is not for azure-private-dns, this was stated in it before and I kept this note.